### PR TITLE
chore: Use nuget config location or default; set dotnet version to 8

### PR DESCRIPTION
**Pull request by nadiiaboichuk**

### CODE REVIEW

File: .github/workflows/pnp-input-api-build.yml [hunk 1]
```diff
@@ -12,8 +12,12 @@ on:
         required: true
       path-to-unit-tests:
         description: |
-          Path to sln or unit tests csproj file
+          Path unit tests project/solution
```
Comment (TurcanStanislav) [inline .github/workflows/pnp-input-api-build.yml:15]: Guess it should be: Path **to** ...

File: .github/workflows/pnp-input-api-build.yml [hunk 2]
```diff
@@ -140,7 +144,7 @@ jobs:
       - name: Setup NuGet sources
         uses: extenda/actions/setup-nuget-sources@v0
         with:
-          config-file: src/nuget.config #check this in all of the APIs
```
Comment (TurcanStanislav) [inline .github/workflows/pnp-input-api-build.yml:143]: I guess the same should be added to `pnp-query-api-build.yml` workflow. The branch name suggests you meant to do this there as well but maybe forgot.

File: .github/workflows/pnp-input-api-build.yml [hunk 3]
```diff
@@ -241,7 +245,7 @@ jobs:
       - name: Setup NuGet sources
         uses: extenda/actions/setup-nuget-sources@v0
         with:
-          config-file: src/nuget.config
+          config-file: ${{ secrets.path-to-nuget-config != '' && secrets.path-to-nuget-config || 'src/nuget.config' }}
```
Comment (TurcanStanislav) [inline .github/workflows/pnp-input-api-build.yml:248]: Is this additional check for empty strings needed? I see [here ](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-a-fallback-value) there is this example: `group: ${{ github.head_ref || github.run_id }}`.

File: .github/workflows/pnp-input-api-build.yml (hunk 1 above)
Comment (nadiiaboichuk) [reply .github/workflows/pnp-input-api-build.yml]: Thanks, I have updated the descriptions

File: .github/workflows/pnp-input-api-build.yml (hunk 3 above)
Comment (nadiiaboichuk) [reply .github/workflows/pnp-input-api-build.yml]: Thanks for suggestion, tested and it works

**Pull request merged**